bench.sh

```diff
@@ -55,7 +55,7 @@ _prepare_install() {
             echo 'git -C /root/.zgen clean -dffx'
             ;;
         zinit )
-            echo 'find /root/.zinit -mindepth 1 -maxdepth 1 ! -name "bin" -exec rm -rf {} \;'
+            echo "for plugin in $(zinit list | sed 's/\x1b\[[0-9;]*[a-zA-Z]//g' | sed 's/ \*//'); do zinit unload $plugin -q; done"
```

Comment:
So this command is run by `hyperfine` before it runs the "install" benchmark and it subprocesses this command, so it can't be shell code. Also does this actually remove the cloned repos?

Reply:
I will recheck if the current command works fine. When checking out the cli this should be the proper way to disable plugins for zinit & zi. I do not get why shell code would not work. Alternatively one could issue sh -c 'THE LOOP'. Unfortunately `zinit list` gives colorized & formatted output and there is no modifier option for that, also there is no command to disable all at once, pretty bad...

Reply:
Check this out: `./bench.sh -k zinit run`. Then execute the old removal command and execute a command. You will get a python 3 error. When you then open a new zsh it will download all the plugins again. It's not just a cache clean what the command does. Propably it's the same with zplug in this benchmark, but did not test it.

Reply:
Just noticed that my command also does not do work properly. Guess these overengineered zinit and zi projects are just broken